src/styles/blocks/content.scss

```
@@ -0,0 +1,16 @@
+.content {
+  display: grid;
+  justify-content: center;
+
+  &__container {
+    width: 944px;
+    height: 862px;
```

```suggestion
    width: 944px;
```
Don't hardcode the height of the container. Let the elements dictate it.
Also, let's calculate the width of the container and other values